frappe/public/js/frappe/views/calendar/calendar.js

```diff
@@ -1,6 +1,12 @@
 // Copyright (c) 2018, Frappe Technologies Pvt. Ltd. and Contributors
 // MIT License. See license.txt
 
+import { Calendar as FullCalendar } from "@fullcalendar/core";
+import dayGridPlugin from "@fullcalendar/daygrid";
+import listPlugin from "@fullcalendar/list";
+import timeGridPlugin from "@fullcalendar/timegrid";
+import interactionPlugin from "@fullcalendar/interaction";
```

Comment:
Can we lazy load this instead OR create a separate bundle for it during build system. 

A small minority uses this feature, not worth adding it in default bundle.